### ISSUE TITLE
Internal Script Error! - opcode #28 (report please).

### ISSUE THREAD

**Issue by Xiuhman** [godotengine/godot#29462]
**Godot version:**
v3.1.1.stable.official

**OS/device including version:**

- OS: Windows 10 64 Bit
- GPU: AMD Radeon RX 550
- CPU: AMD A4-6300 APU
- CPU count: 1
- Logical processors: 2
- Driver name: GLES2
- Thread model: Single-Safe
- VSync: true
- Resolution: (1024, 600)
- Fullscreen: false

**Issue description:**
Internal Script Error! - opcode #28 (report please).
I got this error right after I saved the project while the game was still running.

**Steps to reproduce:**
I've tried multiple times to reproduce it and I only got socket errors and some other weird errors (which had never occurred before).
ERROR: _get_socket_error: Socket error: 10054
At: drivers/unix/net_socket_posix.cpp:190

Images of the main error:
[Image](https://user-images.githubusercontent.com/38364980/58851032-c01dd200-8656-11e9-8f8b-f1f602c60aba.png)
[Image2](https://user-images.githubusercontent.com/38364980/58850188-fd806080-8652-11e9-8ce0-335874b6b9af.png)
Images of errors I got while reproducing this error:
[Image3](https://user-images.githubusercontent.com/38364980/58850988-864ccb80-8656-11e9-8b09-4b1d7a81e1f9.png)
[Image4](https://user-images.githubusercontent.com/38364980/58850990-8cdb4300-8656-11e9-8893-3da1306d1da2.png)

However, almost always the game will crash if I save the project while the game is still running.

**Minimal reproduction project:**
I can't give one and every time I reproduce the error it sends me to a different script, so I'm not sure if it's something with the project, but here's the code where the "Internal Script Error! - opcode #28 (report please)" error happened.
[Enemy_Shooter.zip](https://github.com/godotengine/godot/files/3250665/Enemy_Shooter.zip)

**Comment by bojidar-bg** [comment]
Are you using multiple threads?

**Comment by Xiuhman** [comment]
I'm not sure what that is, but I updated the post to give more information about what I think is what you're saying.

**Comment by akien-mga** [comment]
Can you still reproduce the issue in Godot 3.2.2 or later?

If so, please attach a minimal reproduction project that reliably triggers the issue.

**Comment by Xiuhman** [comment]
I forgot to give an update about this, this error kind of fixed it self with every Godot update but also I think the error was because of reckless loops on my part.

I tried to replicate this with Godot v3.2.2.stable.official and I got nothing, I even tried going back to the version of my game when this issue was happening and I also got nothing.

Since it's basically fixed I will close this issue, I'll keep doing some tests and If I find that this error still exists with a way to reproduce it then I will re open or create a new issue. Thanks.
